ShopifySharp/Infrastructure/Parameterizable.cs

```diff
@@ -58,7 +55,7 @@ public abstract class Parameterizable
         /// <param name="value">The property's value.</param>
         /// <param name="property">The property itself.</param>
         /// <returns>The new parameter.</returns>
-        protected virtual KeyValuePair<string, object> ToSingleParameter(string propName, object value, PropertyInfo property)
+        private KeyValuePair<string, object> ToSingleParameter(string propName, object value, PropertyInfo property)
```

Comment:
Same thing here, this should probably be protected virtual so devs can take control of it where necessary.